doc/source/building/windows.rst

```diff
@@ -8,250 +8,413 @@ Building From Source on Windows
+error messages appeared in your terminal.
 
 Building OpenBLAS
 =================
 
-First, we need to install the software required to build OpenBLAS_, which is the BLAS_
-library that we're going to use. Because the software to build OpenBLAS is different than
-that required to build SciPy and because OpenBLAS takes a long time to build, we're going
-to start building OpenBLAS first and then explain what to do next while the OpenBLAS build
-is running. **Alternatively, if you'd rather download a pre-built OpenBLAS, download the
-one of the** `pre-built zip files`_ **and skip to the Installing OpenBLAS section below.**
-
-Otherwise, install MSYS2 using `these instructions`_ including the `pacman` update
-instructions. Occasionally during the updates the terminal might ask you to close the
-terminal but then might refuse to be closed and hang. If this happens you can kill it via
-Task Manager and continue with the instructions. Make sure to install the correct
-architecture for the SciPy that you want to build (eg. 32 or 64 bit). Now, you have three
-options for opening a terminal which are MSYS2, MINGW (32 or 64 bit). After updating all
-the packages, now we are ready to install some more package bundles that we will need. 
-Open a MSYS2 terminal and type the following depending on the architecture of your 
-choice; run the following for a 32-bit build
+First, we need to install the software required to build OpenBLAS_, which is
+the BLAS_ library that we're going to use. Because the software to build
+OpenBLAS is different than that required to build SciPy and because OpenBLAS
+takes a long time to build, we're going to start building OpenBLAS first and
+then explain what to do next while the OpenBLAS build is running.
+**Alternatively, if you'd rather download a pre-built OpenBLAS, download the
+one of the** `pre-built zip files`_ **and skip to the Installing OpenBLAS
+section below.**. However it is also likely that your version of Windows and
+the compiler you wish to use won't be compatible with what these prebuilt
+binaries produced with. This is still one of the main pain points of building
+for Windows. That's why we will attempt to build our own OpenBLAS.
+
+We start by installing the MSYS2 platform which the OpenBLAS build will take
+place. First download the MSYS2 installer from `msysintaller`_ via choosing
+32 or 64 bit. Make sure to install the correct architecture for the SciPy
+that you want to build (eg. 32 or 64 bit). If you are not sure which to use,
+proceed with 64bit. After the install, in your start menu, you have
+three options for opening a terminal which are MSYS2, MSYS2 MinGW (32/64 bit).
+We are only interested in the latter two depending on which architecture you
+chose. Assuming 64-bit, from now on when we mention MSYS terminal we mean the
+one fired up via `MSYS MinGW 64-bit` option and not the other two.
+
+Choose the one that applies to you and keep running the same command below
```

Comment:
I like the new download link for MSYS2 as there are only two download options now. It also has good install instructions. I would suggest removing our `pacman -Syuu` command below, referring the reader to the install instructions on the page instead.